### PR TITLE
Allow scrolling within dialogs, hidden behind an experiment

### PR DESCRIPTION
**Pull request by ChrisAntaki**
This PR creates an experiment that allows users with small screens to scroll down and see important stuff like the "Already a contributor?" button
image

### CODE REVIEW

File: src/runtime/experiment-flags.js [hunk 1]
```diff
@@ -60,4 +60,9 @@ export const ExperimentFlags = {
    * Experiment flag for guarding changes to fix PayClient redirect flow.
    */
   PAY_CLIENT_REDIRECT: 'pay-client-redirect',
+
+  /**
+   * Enables scrolling within dialogs
```
Comment (mmmicedcoffee) [inline src/runtime/experiment-flags.js:65]: nit: missing period at the end of this comment. 

it doesn't seem to be consistently applied in this file, but I've always used punctuation, capitalization, etc, in my JSDoc comments.

File: src/ui/activity-iframe-view-test.js [hunk 2]
```diff
@@ -97,6 +106,30 @@ describes.realWin('ActivityIframeView', {}, (env) => {
       expect(activityIframePort.whenReady).to.have.been.calledOnce;
     });
 
+    it('should disallow scrolling within dialogs', async () => {
```
Comment (mmmicedcoffee) [inline src/ui/activity-iframe-view-test.js:109]: nit: can omit "should" for brevity, so the test description would read something like "disallows scrolling within dialogs"

File: src/ui/activity-iframe-view-test.js [hunk 3]
```diff
@@ -97,6 +106,30 @@ describes.realWin('ActivityIframeView', {}, (env) => {
...
+        win,
+        activityPorts,
+        src,
+        activityArgs
+      ).getElement();
+
+      expect(iframe.scrolling).to.equal('no');
+    });
+
+    it('should (optionally) allow scrolling within dialogs', async () => {
```
Comment (mmmicedcoffee) [inline src/ui/activity-iframe-view-test.js:120]: same here - this could be: "allows scrolling within dialogs behind an experiment"

**Review by mmmicedcoffee — APPROVED** [review]

File: src/runtime/experiment-flags.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/runtime/experiment-flags.js]: Added!

File: src/ui/activity-iframe-view-test.js (hunk 2 above)
Comment (ChrisAntaki) [reply src/ui/activity-iframe-view-test.js:109]: Good call

File: src/ui/activity-iframe-view-test.js (hunk 3 above)
Comment (ChrisAntaki) [reply src/ui/activity-iframe-view-test.js]: That's clearer 👍

**Pull request merged**